test/extended/operators/resources.go

```diff
@@ -144,6 +144,9 @@ var _ = g.Describe("[sig-arch] Managed cluster", func() {
 					// Pods may not set limits
```

Comment:
I'm not sure why these tests are under the `operators` test package. From what I can see, they don't look related to the operator-framework. Were these tests placed here by mistake?

Reply:
Looks like it was originally added in #25959